README.rst

```diff
@@ -74,14 +89,70 @@ Each editor defined in ``WAGTAILADMIN_RICH_TEXT_EDITORS`` is a dictionary with 2
 -  ``WIDGET`` is a mandatory string set to the widget to use
    -  should always be set to ``wagtaildraftail.widgets.DraftailTextArea`` (or a subclass of it) to work with Draft.js content
 -  ``OPTIONS`` is a dictionary which follows the format of `Draftail configuration options <https://github.com/springload/draftail#usage>`_.
-   -  Draftail options which are JavaScript values are hydrated at runtime in ``client/wagtaildraftail.js``
+   -  Draftail options which are JavaScript values are hydrated at runtime in ``client/wagtaildraftail.js``. Alternatively, on Wagtail 1.12 and above, a ``features`` list can be passed within the ``OPTIONS`` dict in place of a full Draftail configuration.
 
 **WARNING:** The ``type`` key for ``blockTypes``, ``inlineStyles`` and ``entityTypes`` shouldn’t be changed. It is what defines how content is rendered, and is saved as a JSON blob in the database which would make migrations really painful.
 
 **WARNING:** All the blocks/styles/entities defined in the editor config should have been configured to render properly in the `exporter config <#exporter-configuration>`_.
 
 Here is a sample configuration file. This should live in your Django settings.
 
+For Wagtail 1.12.x and above:
+
+.. code:: python
+
+    from draftjs_exporter.constants import BLOCK_TYPES, ENTITY_TYPES
+    from draftjs_exporter.defaults import BLOCK_MAP
+
+    WAGTAILADMIN_RICH_TEXT_EDITORS = {
+        'default_draftail': {
+            'WIDGET': 'wagtaildraftail.widgets.DraftailTextArea',
+        },
+
+        'format_and_link': {
+            'WIDGET': 'wagtaildraftail.widgets.DraftailTextArea',
+            'OPTIONS': {
+                'features': ['link', 'bold', 'italic'],
```

Comment:
This is a valid concern, but it's an issue with the concept of rich text features as a whole, rather than the Draftail implementation of them, so I'd say it's out of scope for this PR.

In theory, changes to the features list can affect the meaning of the data, not just the behaviour of the editor (it's possible that some element within the data could be interpreted in several different ways depending on which features were active) so we should indeed be keeping track of it within migrations. In practice, this "multiple interpretations of the data" effect can't really happen at the moment - mainly due to the `|richtext` filter, which receives a string and has to transform it into front-end HTML without any knowledge of which field the string came from. So, for now, tracking the state of the features list is only a theoretical concern.

(#61 shows there might be a way out of this: _if_ the `|richtext` filter can be eliminated in favour of using `RichText` value objects everywhere, then we can set up `RichText` objects to remember their own feature lists. But let's not get ahead of ourselves :-) )